Oware/src/Player.java

```diff
@@ -2,10 +2,45 @@
 public class Player
 {
 
-	public static void main(String[] args)
+	private int playerStorehouse;
+	private int opponentStorehouse;
```

Comment:
I see, we have one player object to represent both players...  That is a bit scary considering the playerstorehouse is still the playerstorehouse even as the player changes.